agent/rpm/pbench-agent.spec.j2

```diff
@@ -63,41 +72,25 @@ mkdir -p %{buildroot}/%{installdir}
 cp -a agent/* %{?buildroot}/%{installdir}/
 
 %pre
-# this RPM conflicts with a configtools RPM, but we may have a Pypi
+# This RPM conflicts with a configtools RPM, but we may have a PyPI
 # configtools installed: zap it.
-%if 0%{?rhel} == 7
-scl enable rh-python38 -- bash -c 'if pip3 show configtools > /dev/null 2>&1 ;then pip3 uninstall -y configtools ;fi'
-%endif
-
-%if 0%{?rhel} == 8 || 0%{?fedora} >= 31
 if pip3 show configtools > /dev/null 2>&1 ;then pip3 uninstall -y configtools ;fi
-%endif
 
 %post
-
-%if 0%{?rhel} == 7
-sed -i 's/^#\(source scl_source enable rh-python38\)/\1/' /%{installdir}/base
-%endif
-
 # Install python dependencies
-%if 0%{?rhel} == 7
-scl enable rh-python38 -- bash -c "pip3 --no-cache-dir install --prefix=/%{installdir} -r /%{installdir}/requirements.scl" > /%{installdir}/pip3-install.log
-%endif
-%if 0%{?rhel} == 8 || 0%{?fedora} >= 31
-pip3 --no-cache-dir install --prefix=/%{installdir} -r /%{installdir}/requirements.f3x > /%{installdir}/pip3-install.log
-%endif
+pip3 --no-cache-dir install --prefix=/%{installdir} -r /%{installdir}/requirements.txt > /%{installdir}/pip3-install.log
```

Comment:
I need a memory jog: we manufacture different `requirements.txt` files for the different distros, based on the mapping table - correct? And that change has already been merged - correct?

Reply:
Ah, yes; I think that's correct, and I'd forgotten it. This has gone through so many permutations, my brain is still bouncing.

Reply:
I'm glad I'm not the only one :-)

Reply:
I have addressed this in the commit updated commit message and in the description.